ArduCopter/radio.cpp

```diff
@@ -143,15 +143,13 @@ void Copter::set_throttle_and_failsafe(uint16_t throttle_pwm)
 
         // check for 3 low throttle values
         // Note: we do not pass through the low throttle until 3 low throttle values are received
-        failsafe.radio_counter++;
-        if( failsafe.radio_counter >= FS_COUNTER ) {
+        if( ++failsafe.radio_counter >= FS_COUNTER ) {
```

Comment:
I think this pattern is harder to understand.

Reply:
I agree, I'm not a fan of this change I'm afraid.